examples/UKCore-Condition-Extension-CodingSCTDescId-SN-IdealWeight-Ex.xml

```diff
@@ -0,0 +1,26 @@
+<Condition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<id value="UKCore-Condition-Extension-CodingSCTDescId-SN-IdealWeight-Ex"/>
+	<code>
+	  <coding>
+			<extension url="https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-CodingSCTDescId">
```

Comment:
shouldn't this use the 

 <extension url="http://hl7.org/fhir/StructureDefinition/coding-sctdescid">
                <valueid>
                    <extension url="https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-CodingSCTDescDisplay">
                    
 format of extensions